docs/custom_framework.md

```diff
@@ -1,39 +1,39 @@
 ## 🎎 Custom Framework Config
 
-If you are using an i18n framework that does not have built-in support by this extension or you are using a custom i18n implementation, you can always make this extension support for your framework.
+If you are using an i18n framework that does not have built-in support by this extension, or you are using a custom i18n implementation, you can always make this extension support for your framework.
 
 Create file `.vscode/i18n-ally-custom-framework.yml` then copy and paste the configs below.
 
 ```yaml
 # .vscode/i18n-ally-custom-framework.yml
 
-# An array of string which contains Language Ids defined by vscode
+# An array of strings which contain Language Ids defined by VS Code
 # You can check avaliable language ids here: https://code.visualstudio.com/docs/languages/overview#_language-id
 languageIds:
   - javascript
   - typescript
   - javascriptreact
   - typescriptreact
 
-# An Array of regex to find the keys usage. **The key should captured in the first match group**.
-# You should unescape regex string in order to fit in YAML file
-# for that, you can use https://www.freeformatter.com/json-escape.html
+# An array of RegExs to find the key usage. **The key should be captured in the first match group**.
+# You should unescape RegEx strings in order to fit in the YAML file
+# To help with this, you can use https://www.freeformatter.com/json-escape.html
 keyMatchReg:
-  # The following examples show how to detect `t("your.i18n.keys")`
+  # The following example shows how to detect `t("your.i18n.keys")`
   # the `{key}` will be placed by a proper keypath matching regex,
   # you can ignore it and use your own matching rules as well
   - "[^\\w\\d]t\\(['\"`]({key})['\"`]"
 
 
-# An Array of string contains refactor templates.
+# An array of strings containing refactor templates.
 # The "$1" will be replaced by the keypath specified.
-# Optional, uncomment the following two lines to use
+# Optionally, uncomment the following two lines to use
```

Comment:
I think `Optional` is more like a statement. Maybe we should change the comma to period.

Reply:
A colon would probably be best. "Optional: uncomment the following two lines to use"

Reply:
Cool 👍

Reply:
Updated in latest commit.